tdrs-frontend/src/actions/requestAccess.js

```diff
@@ -13,7 +13,7 @@ export const requestAccess =
     dispatch({ type: PATCH_REQUEST_ACCESS })
     try {
       const URL = `${process.env.REACT_APP_BACKEND_URL}/users/set_profile/`
-      const user = { first_name: firstName, last_name: lastName, stt: { id } }
+      const user = { first_name: firstName, last_name: lastName, stt: id }
```

Comment:
We just need to pass the ID by itself now, but the returned payload to the state reducer will still be the entire object so nothing else needs to change on the front end.